This patch deactivates defining the "drbd_genl_multicast_group" function,
which gets overridden.


diff --git i/drbd/linux/genl_magic_func-genl_register_mc_group.h w/drbd/linux/genl_magic_func-genl_register_mc_group.h
index e042f165..1d46d84d 100644
--- i/drbd/linux/genl_magic_func-genl_register_mc_group.h
+++ w/drbd/linux/genl_magic_func-genl_register_mc_group.h
@@ -4,7 +4,7 @@ static struct genl_multicast_group					\
 CONCAT_(GENL_MAGIC_FAMILY, _mcg_ ## group) __read_mostly = {		\
 	.name = #group,							\
 };									\
-static int CONCAT_(GENL_MAGIC_FAMILY, _genl_multicast_ ## group)(	\
+static int CONCAT_(GENL_MAGIC_FAMILY, _genl_multicast_ ## group ## _UNUSED)(	\
 	struct sk_buff *skb, gfp_t flags)				\
 {									\
 	unsigned int group_id =						\
